### PR TITLE
Add German translation

### PR DESCRIPTION
**Pull request by wlhlm**
This adds an initial German translation.

Feedback from native speakers appreciated!

### CODE REVIEW

File: resources/assets/storagedrawers/lang/de_DE.lang [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+storageDrawers.config.prop.itemRenderType.tooltip=Rendere Items in 3D oder flach (2D).
+storageDrawers.config.prop.enableIndicatorUpgrades=Erlaube Indikatorupgrades
+storageDrawers.config.prop.enableStorageUpgrades=Erlaube Storageupgrades
+storageDrawers.config.prop.enableLockUpgrades=Erlaube Schubfachverschließung
+storageDrawers.config.prop.enableShroudUpgrades=Erlaube Verbergen
+storageDrawers.config.prop.enableVoidUpgrades=Erlaube Voidupgrade
+storageDrawers.config.prop.enableCreativeUpgrades=Erlaube Kreativupgrade
+storageDrawers.config.prop.enableSortingUpgrades=Erlaube Sortierungsupgrade
+storageDrawers.config.prop.enableSortingUpgrades.tooltip=Refined Relocation muss geladen seien.
+storageDrawers.config.prop.renderStorageUpgrades=Render Speicherupgrades
```
Comment (Vexatos) [inline resources/assets/storagedrawers/lang/de_DE.lang]: How about "Zeige Speicherupgrades" or "Speicherupgrades zeigen" or "Speicherupgrades darstellen"?

Comment (wlhlm) [inline resources/assets/storagedrawers/lang/de_DE.lang]: :+1:

**Comment by wlhlm** [comment]
Updated.

Thanks a lot for having a look at this (and all my other translation PRs)!

**Comment by Vexatos** [comment]
@wlhlm I myself am glad that there's finally another good translator caring about German translations :+1:

**Comment by jaquadro** [comment]
Thanks for the German translation and feedback.

**Pull request merged**